Gemfile

```diff
@@ -18,7 +18,7 @@ else
   gem 'gds-api-adapters', "~> 47.9.1"
 end
 
-gem "gds-sso", "13.0.0"
+gem "gds-sso"
```

Comment:
Should this be tagged to `~> 13.4` so that it doesn't get bumped into breaking changes unexpectantly?